CocoaPods.md

```diff
@@ -72,18 +72,31 @@ my-plugin/
         └── Podfile
 ```
 
+Podfile:
+```
+pod 'GoogleMaps'
+```
+
+## Install the Plugin
+
 Next, install the plugin:
 
-```
+```bash
 tns plugin add ../my-plugin
 ```
 
-## Build the project
+> **NOTE:** By default, all CocoaPods are installed as [shared frameworks](https://blog.cocoapods.org/CocoaPods-0.36/) in order all symbols to be available at runtime. Shared frameworks are supported on iOS 8+ only, so the iOS deployment target of your app will be upgraded to a minimum of that.
```

Comment:
`NOTE: Installing CocoaPods sets the deployment target of your app to iOS 8, if not already set to iOS 8 or later. This change is required because CocoaPods are installed as shared frameworks to ensure that all symbols are available at runtime.`, as @ikoevska's suggested.